modules/effects/migrations/18_0_0-beta/index.ts

```diff
@@ -53,6 +71,7 @@ export function migrateConcatLatestFromImport(): Rule {
         .map((element) => element.name.getText())
         .join(', ');
 
+      const changes: Change[] = [];
```

Comment:
If I understand correctly this is the "real" fix as we're creating a new collection for each file, correct?

Reply:
Exactly, that's the main one. I see the others as edge cases.